docs/docs/about.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: About
+permalink: /about
+nav_order: 2
+---
+
+## About OCI
+
+The Open Container Initiative was established in June 2015 by 
+[Docker](https://en.wikipedia.org/wiki/Docker_(software)), one of the many
+early leaders of the container industry that provided an intuitive API to interact
+with [linux containers](https://en.wikipedia.org/wiki/Container_(virtualization)).
+
+### Challenges for OCI
```

Comment:
I've rephrased it so that the "defining standards for..." is the focus.

Reply:
That's weird, I must have somehow undone my changes. This "squash commit" thing makes it really hard to contribute :/

Reply:
I disagree about removing this section - I had originally rephrased it so that defining standards is the focus. I'll rephase more so it's not a bulleted list.